dev/create-release/vote.tmpl

```diff
@@ -0,0 +1,64 @@
+Please vote on releasing the following candidate as Apache Spark version {version}.
+
+The vote is open until {deadline} and passes if a majority of at least 3 +1 PMC votes are cast.
```

Comment:
nit: personally I find `3 PMC +1 votes` more clear

Reply:
The rules are a little bit more complicated than that. Updated.